index.html

```diff
@@ -123,97 +117,75 @@
     <dt>`entryType`</dt>
     <dd>This attribute MUST return the DOMString "`server`".</dd>
     <dt>`startTime`</dt>
-    <dd>This attribute MUST return value 0.</dd>
+    <dd>This attribute MUST return value 0. Because there can be no guarantee of clock synchronization between client, server, and intermediaries, it is impossible to map a meaningful `startTime` onto the clients timeline.</dd>
     <dt>`duration`</dt>
     <dd>
-      This attribute MUST return a <a data-cite="hr-time-2#idl-def-domhighrestimestamp">`DOMHighResTimeStamp`</a> [[!HR-TIME-2]] that contains the server-specified metric value, or value 0.
+      This attribute MUST return a number that contains the server-specified metric value, or value 0.
```

Comment:
See my other comment, but I'm not sure `name`, `entryType`, `startTime` and even `duration` make much sense here now that we've coupled ST with RT.